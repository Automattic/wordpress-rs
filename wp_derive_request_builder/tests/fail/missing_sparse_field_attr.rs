#[derive(wp_derive_request_builder::WpDerivedRequest)]
#[Namespace("/wp/v2")]
enum UsersRequest {
    #[contextual_get(url = "/users", output = std::vec::Vec<SparseUser>)]
    List,
}

fn main() {}
