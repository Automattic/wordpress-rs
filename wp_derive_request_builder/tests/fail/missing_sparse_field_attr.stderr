error: Missing #[SparseField(_field_type_)]
 --> tests/fail/missing_sparse_field_attr.rs:2:1
  |
2 | enum UsersRequest {
  | ^^^^
